src/sanitiser.test.js

```diff
@@ -6,57 +6,58 @@ describe('sanitiser', () => {
       const configuration = sanitiseGlobalConfiguration({});
       expect(configuration).toEqual({
```

Comment:
These config tests seem like they could be snapshot tests

Reply:
What do you mean by snapshot tests?

Reply:
Any serialisable bit of data can be put in a snapshot. So rather than have this big manually created object to check the config against why not use snapshot testing? 
``` 
expect(configuration).toMatchSnapshot();
```

Reply:
you'll get something like this
```
// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sanitiser sanitise global configuration sanitise if screenshots config provided 1`] = `
Object {
  "debug": false,
  "isUpdate": false,
  "mismatchThreshold": 0.01,
  "puppeteer": Object {
    "args": Array [],
    "dumpio": false,
    "executablePath": undefined,
    "headless": true,
    "ignoreHTTPSErrors": false,
    "slowMo": 0,
    "timeout": 30000,
  },
  "saveDifferencifiedImage": true,
  "screenshots": "./somewhere",
  "testReports": "./differencify_report",
}
`;
```

Reply:
You can even run the tests once with both the snapshot test and the old test and make sure things match.

Reply:
Awesome. Didn't know that. will do

Reply:
Done